vividus-util/src/main/java/org/vividus/util/freemarker/FreemarkerProcessor.java

```diff
@@ -50,6 +51,7 @@ public FreemarkerProcessor(Class<?> resourceLoaderClass, String templatePath)
         configuration = new Configuration(Configuration.VERSION_2_3_30);
         configuration.setClassForTemplateLoading(resourceLoaderClass, templatePath);
         configuration.setDefaultEncoding(StandardCharsets.UTF_8.toString());
+        configuration.setOutputFormat(XMLOutputFormat.INSTANCE);
```

Comment:
Define the format at a template level 
https://freemarker.apache.org/docs/ref_directive_outputformat.html